tests/test_rest_api.py

```diff
@@ -0,0 +1,49 @@
+import requests
```

Comment:
@GeetikaBatra tests have to be rewritten. These tests and sample payload are not valid for this API server. Please keep a placeholder test function until we implement the API functionalities.

Reply:
Okay.